src/components/PrayerTimingsTable.tsx

```diff
@@ -98,14 +104,14 @@ const PrayerTimingsTable = (props: {
                 >
                   <TableCell
                     sx={{
-                      fontWeight: compareFormattedTimes(row.timing) ? 300 : 700
+                      fontWeight: (compareFormattedTimes(row.timing) && 0<numBolded && numBolded<3) ? 700 : 300
```

Comment:
@shaji618 I feel like this should work but it doesn't. My best guess is that the formatted times are not compared the way I think they are. This results in nothing being bolded instead of the first matching row as I intended.

Reply:
I kinda wanna talk through this one.  The API returns what, in my opinion, is some pretty shitty data, and we suffer from this since the data is _only_ available pre-formatted for time zone.

In this case, I think we need to re-think how we're using the ugly data that we're getting back.  In my opinion, we should probably pre-process before rendering at all.  So we take our response from the API and then compare the entire data set to the current time.  Thus we should be able to grab the first one we want and render just that...instead of the lazy, CSS-reliant solution that I put in place just to get the ball moving.

Let me know if you wanna pair on this bc I'm not entirely sure my thought process is right either.

Reply:
Yeah pairing could be a good approach here for sure. 

> hus we should be able to grab the first one we want and render just that...instead of the lazy, CSS-reliant solution that I put in place just to get the ball moving.

Do you mean we only display the upcoming prayer session and don't display past or future ones?

Reply:
I think I wrote my last comment when I was a little tired.  What I meant to say is that I believe that (my "working" branch kinda proves this out) we should be taking an additional step before we paint -- instead of grabbing _all_ the rows that should be bold, we only grab the one we want -- and then we have fewer moving parts in our return jsx.  Hopefully that helps you along/explains my thinking a bit better.

Thinking more about it, it might be good for you to take a peek at my "good" branch:

1. You might have some comments/notice some things that I missed
2. I'll probably need to go ahead and push my change in a couple of days

LMK when you wanna pair, and I'm happy to do it

Reply:
Yeah I'd love to take a look at your branch when able. I think I could pair sometime this weekend, or if not then Eastern Time lunch on Monday would work. The next few days I'm slammed with interviews as I explore other options that might be better than my current place, though I'm questioning that as well. This turned into a mini therapy rant 😂  Let me know when might work for you to pair.

Reply:
Oh wait it looks like you may have already merged it. Checking now